CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Changelog
+## 1.4.1
```

Comment:
I'll put version number on including the changes in batch, so can you remove this change?

Reply:
Done! ;)